tests/helpers.py

```diff
@@ -2101,3 +2101,20 @@ def modify_osd_replica_count(resource_name, replica_count):
     params = f'{{"spec": {{"replicas": {replica_count}}}}}'
     resource_name = '-'.join(resource_name.split('-')[0:4])
     return ocp_obj.patch(resource_name=resource_name, params=params)
+
+
+def check_local_volume():
+    """
+    Function to check if Local-volume is present or not
+    {To be used for BM Cluster}
```

Comment:
This could potentially be used for any platform using LSO so I don' t think we need this line.